src/main/java/Duke.java

```diff
@@ -1,10 +1,54 @@
+import java.util.Scanner;
```

Comment:
Good practice listing the imported class explicitly instead of implicitly e.g. import java.util.*